### PR TITLE
Add standardized/normalized data to catalog

### PR DESCRIPTION
**Pull request by tdrwenski**
This PR would add standardized and normalized versions of one type of satellite data (Cloud top temperature) and GFS Quarter Degree Analysis to the thredds catalog. This is already deployed on [thredds-test](https://thredds-test.unidata.ucar.edu/thredds/catalog/catalog.html) if you want to see what it looks like. One difference between the preprocessed GFS data and the raw data is that for the preprocessed data we are using a datasetScan because NcML does not (currently) work with featureCollections.

Note that we only have one catalog for both thredds and thredds-test so adding this catalog means we either need to make a distinction between the catalogs of the two servers or else deploy on the thredds server as well.

### CODE REVIEW

**Comment by dopplershift** [comment]
Are we using threddsDev anywhere? If so, then adding thredds-test back that inherits idd and the main thredds, then adds this catalog would be pretty easy.

**Comment by tdrwenski** [comment]
> Are we using threddsDev anywhere? If so, then adding thredds-test back that inherits idd and the main thredds, then adds this catalog would be pretty easy.

I guess threddsDev is used for the threads-dev server. Can definitely add thredds-test as a separate folder/top catalog if you prefer that over offering the preprocessed data on both servers upon our next release.

**Comment by dopplershift** [comment]
I forgot that thredds-dev still existed.

I don't necessarily have a preference; not sure whether this is something we want to offer in production or not.

**Comment by haileyajohnson** [comment]
I don't know if we want to offer this in production (at least yet), but if we add a separate test directory we want to make sure it gets everything that in thredds. It can be a superset of it but it needs to have everything we offer there

**Comment by tdrwenski** [comment]
As discussed, we will separate the configs for thredds and thredds-test, with the latter having the preprocessed data.

I am not so familiar with this repo, can you check that I have included the correct files in `threddsTest`? There are some files I see that I am not sure what they are used for. Should I also verify by deploying this that it thredds-test still works correctly (if so, can someone help me with this)?

File: threddsTest/build.info [hunk 1]
```diff
@@ -0,0 +1,2 @@
+../wmsConfig.xml
+../idd
```
Comment (dopplershift) [inline threddsTest/build.info:2]: This should pull in all of what's used in the idd and thredds directory (in order), then just need to override the top-level catalog so the pre-processed catalog/notebook can be included:
```suggestion
../idd
../thredds
../wmsConfig.xml
```

File: threddsTest/threddsConfig.xml
Comment (dopplershift) [inline threddsTest/threddsConfig.xml:1]: You should be able to remove this file after "inheriting" the `thredds/` directory. Unless you made some change here that I missed?

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a few changes to make this better be just a small departure from the production thredds.ucar.edu config. Deployment on thredds-test *should* be as simple as downloading the newly created zip from the archive server and unzipping to the appropriate spot in `/opt/tds-test`.

File: threddsTest/build.info (hunk 1 above)
Comment (tdrwenski) [reply threddsTest/build.info:2]: thanks, I see how that works now-- that is nicer. Fixed in 2edb0e56f7214270f90be90b5eaa79ad26b4230c

File: threddsTest/threddsConfig.xml
Comment (tdrwenski) [reply threddsTest/threddsConfig.xml:1]: removed in 2edb0e56f7214270f90be90b5eaa79ad26b4230c

**Review by dopplershift — APPROVED** [review]
Looks good. Let's see how it goes on the server...

**Pull request merged**